src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/NLogLogging/NLogWrapper.cs

```diff
@@ -77,6 +91,11 @@ private void DecorateLogMessage(object logEvent, Type logEventType, IAgent agent
                 return;
             }
 
+            if (formattedMessage.Contains("NR-LINKING"))
```

Comment:
Maybe we could make this a function in `LoggingHelpers` so we don't need to know about the token here?

Reply:
That seems like it wouldn't be a bad idea,